playbooks/byo/openshift-cluster/redeploy-certificates.yml

```diff
@@ -14,7 +14,36 @@
   gather_facts: no
   tasks:
   - include_vars: ../../byo/openshift-cluster/cluster_hosts.yml
+  - set_fact:
+      openshift_deployment_type: "{{ deployment_type }}"
```

Comment:
Ah, that's right...  We probably need to start exposing openshift_deployment_type in the inventory, and just mapping deployment_type to openshift_deployment_type for legacy use.